frank-doc-doclet/src/test/java/org/frankframework/frankdoc/testtarget/ibisdocref/ChildTarget.java

```diff
@@ -18,21 +9,6 @@ public void otherMethod(String value) {
 	public void setAttributeWithIbisDocRefReferringJavadoc(String value) {
```

Comment:
Dit gaat ook nog over IbisDoclRef .. klan dit weg?

Reply:
Er zijn nog redelijk wat test functies die een naam hebben die gerelateerd is aan het ibis systeem. Al deze namen aanpassen kost best veel moeite, omdat de testen niet altijd even transparant zijn. Ik vind het persoonlijk zonde van de tijd om ze aan te passen.